### DISCUSSION TITLE
Screenshot: add new button

### DISCUSSION THREAD

**Discussion by accron-1** [linuxmint/discussions#347] [Ideas]
Currently (on Cinnamon) after a screenshot has been taken, three buttons are displayed: **Cancel**, **Copy to Clipboard**, **Save**.
If one wants to open the image immediately after the screenshot has been taken one has to first save the file, open Nemo and then locate the file.

I think it would be very useful to add another button, **Open**, along with a dropdown list with all the applications that could open the image (the same application list that is displayed in 'Open With' in Nemo in the right click menu). The dropdown should preselect the system default app for images.
If **Open** is clicked, the image should automatically be saved (as if the user had clicked 'Save' with all the suggested default values), and then open the image with the selected application in the dropdown.

**Comment by goyalyashpal** [comment]
duplicate of 
* #524
